core-nonfree/story-api-nonfree/src/main/java/edu/cmu/cs/dennisc/nebulous/Manager.java

```diff
@@ -82,15 +82,21 @@ public class Manager {
 
   public static native void setDebugDraw(boolean debugDraw);
 
+  private static long lastNotifiedAboutAssets;
```

Comment:
should this be initialized?

Reply:
The default value for a long is 0 so I did not.

Reply:
But a quick review of Alice code shows initializing to 0 is more frequent, so I have joined in.